projects/cashmere/src/lib/sass/stepper.scss

```diff
@@ -0,0 +1,185 @@
+$track-space: 4rem;
+$isolated-disk-size: 2rem;
+$track-height: 0.5rem;
+
+@mixin hc-disk-border($border-color) {
+    box-shadow: 0px 0px 0px 3px $border-color;
+}
+
+@mixin hc-step-arrow($top-bottom, $right) {
+    @if $right {
+        border-right: $track-space * 0.05 solid $right;
+    }
+    border-left: $track-space / 4 solid transparent;
+    border-top: $track-space / 2 solid $top-bottom;
+    border-bottom: $track-space / 2 solid $top-bottom;
+    display: block;
+    height: 0px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 2;
+}
+
+@mixin hc-stepper-bg-color($color) {
+    background-color: $color;
```

Comment:
Getting philosophical for a moment. I have to wonder, is this a useful mixin? If the purpose of the mixins is to allow people to apply Cashmere styles without using the full Angular component, then a mixin like this doesn't help much--if they want a "green" stepper, for example, this still requires them to dig around in source code to find the appropriate color variable. I'll approve for now, but maybe this is something we should mull over.

Reply:
Yep - I totally hear what you're saying.  And after this sprint, I'd love for you to give this one a look over.  Because I think I've followed the letter of the law on this mixin, but not really the spirit of it.  My brain was hurting last night just getting it this far though.  But now that it's at least up to the minimum standards - I agree we could do better.